### PR TITLE
Add some additional reliable tumblr classes to documentation

### PR DESCRIPTION
**Pull request by lorrocha**

### CODE REVIEW

File: docs/Tumblr-Classes.md [hunk 1]
```diff
@@ -11,21 +11,27 @@ These are the stylesheet classes used by Tumblr on their pages.
...
+* `data-type` for post Type of the post
 * `data-following-tumblelog` if following that user
 * `data-tumblelog-name` for owner of the post
+* `data-root-id` for the post ID of the original post
+* `data-json` the contents of the post in json format
+* `data-is-animated` whether the original photoset or video has animation
 
 ### Post Types
 
+* .is_regular
```
Comment (nightpool) [inline docs/Tumblr-Classes.md]: I'm not sure regular and reblog are posts types in the same way that the rest of these are... They should probably go in "others" instead.

They definitely need more explanation if we're going to document them usefully.

**Comment by nightpool** [comment]
Hey, are you interested in updating this with the feedback I gave? If not, that's fine, but I'd like to resolve the PR one way or the other.

**Comment by lorrocha** [comment]
Yes, sorry for the extended wait! I kind of forgot that this was still on my plate, haha. I made a few adjustments and added another class I noticed, so it's good for another round of reviews.

**Comment by nightpool** [comment]
Thanks a ton! made a few small tweaks: post-json should probably also be documented, but that's probably a separate PR

LGTM @homu r+

**Pull request merged**